plugins/modules/clusters.py

```diff
@@ -108,7 +108,14 @@ def run_module():
             module.fail_json(msg=f"Error deleting cluster_id: {module.params.get('cluster_id')}", **result)
         
         result = dict(changed=True, clusters=[])
-
+    # Retrieve list of OpenShift supported versions
+    elif module.params.get('action') == "list-openshift-versions":
```

Comment:
are you planning to have a versions module in which case, the below can be part of it  as well
```
[/v2/component-versions](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openshift/assisted-service/master/swagger.yaml#/versions/v2ListComponentVersions)
[/v2/release-sources](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/openshift/assisted-service/master/swagger.yaml#/versions/v2ListReleaseSources)
```